<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMissingEpisodes</class>
 <widget class="QWidget" name="CMissingEpisodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Missing Episodes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="dataSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="serversLabel">
         <property name="text">
          <string>Servers:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select Shows to Filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="unselectAll">
         <property name="text">
          <string>Unselect All</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="mediaSummaryLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Media Summary:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeWidget" name="showsFilter">
         <column>
          <property name="text">
           <string notr="true">Show</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Min Season</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Max Season</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QTreeView" name="servers">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="selectAll">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
